pipeline/scheduled-pipeline-executor.groovy

```diff
@@ -0,0 +1,58 @@
+/*
+    Pipeline script for executing weekly scheduled test suites
+*/
+// Global variables section
+
+// Server to get recipe file contents
+def SSHServer = "ssh 10.245.4.89"
+
+// Recipe file path details
+def recipeFile = "RHCEPH-*.yaml"
+def recipeFileDir = "/data/site/recipe"
+
+// Default job parameters
+def buildType = "${params.buildType}" ?: "tier-1"
+def overrides = "${params.overrides}" ?: "{}"
+def tags = "${params.tags}" ?: ""
+def tagsList = tags.split(',') as List
+def nodeName = "centos-7"
+if ("ibmc" in tagsList) {
+    nodeName = "agent-01"
+}
```

Comment:
I believe this is specific to IBM due to the setting of the default value of `SSHServer`

hence can we rename the script to `ibm-scheduled-pipeline-trigger.groovy`

Reply:
Will add common logic for both IBM and PSI cloud in same script.